frontend/lib/src/hostComm/HostCommunicationManager.test.tsx

```diff
@@ -38,7 +38,7 @@ describe("HostCommunicationManager messaging", () => {
   let dispatchEvent: (type: string, event: Event) => void
   let originalHash: string
 
-  let setAllowedOriginsFunc: jest.SpyInstance
+  let setHostConfigFunc: jest.SpyInstance
```

Comment:
nit: should this still be `setAllowedOriginsFunc`? The function spied on below is still `setAllowedOrigins`

Reply:
yep, indeed. Fixed 👍